packages/pds-ember/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@hashicorp/pds-ember",
+  "version": "0.0.3",
+  "description": "Ember addon for building Structure-styled UIs",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hashicorp/structure.git"
+  },
+  "keywords": [
+    "ember-addon"
+  ],
+  "author": "Ryan Johnson <rhino.citguy@gmail.com>",
+  "contributors": [],
+  "license": "MPL-2.0",
+  "bugs": {
+    "url": "https://github.com/hashicorp/structure/issues"
+  },
+  "homepage": "https://github.com/hashicorp/structure#readme",
+  "dependencies": {},
+  "publishConfig": {
+    "access": "public"
```

Comment:
Without this, the package will be private to `@hashicorp` NPM org members, only.

Reply:
This can theoretically be moved to `packages/pds-ember/.npmrc` as ...

```
access = "public"
```